src/client/src/components/pages/participants/viewParticipants/index.jsx

```diff
@@ -27,9 +24,77 @@ export default class ViewParticpants extends Component {
   clear = () => {
     this.setState({ search: "" });
   };
+
+  onDelete = id =>{
+    swal({
+      title: "Warning",
+      text: "Are you ready that you want to delete this participant ?",
```

Comment:
Are you <code>sure</code> not <code>ready</code>

Reply:
done